lib/node/node.go

```diff
@@ -0,0 +1,28 @@
+package sebaknode
+
+import (
+	"boscoin.io/sebak/lib/common"
+
+	"github.com/stellar/go/keypair"
+)
+
+type Node interface {
+	Address() string
+	Keypair() *keypair.Full
+	SetKeypair(*keypair.Full)
+	Alias() string
+	SetAlias(string)
+	Endpoint() *sebakcommon.Endpoint
+	Equal(Node) bool
+	DeepEqual(Node) bool
+	GetValidators() map[string]*Validator
+	AddValidators(validators ...*Validator) error
+	HasValidators(string) bool
+	RemoveValidators(validators ...*Validator) error
```

Comment:
`RemoveValiators()` and it's relatives is only used for `LocalNode`, not for `Validator`, It would be better to be removed from `Node` interface. If you have any usage this methods, ignore this comment.

Reply:
@spikeekips I agree with you. Then these methods have no reason to exists, right?

```
GetValidators() map[string]*Validator
AddValidators(validators ...*Validator) error
HasValidators(string) bool
RemoveValidators(validators ...*Validator) error
```